database/mongoose.ts

```diff
@@ -0,0 +1,27 @@
+const mongoose = require('mongoose')
+
+import { Request, Response, NextFunction } from 'express'
+import { Schema, model, connect } from 'mongoose'
+
+import Trip from '../models/trip'
+
+mongoose.connect('mongodb+srv://jmarchant:XyIWQYpZuJwGkKbN@default.xb0hjgi.mongodb.net/goblinShark?retryWrites=true&w=majority').then(() => {
```

Comment:
In a recent commit I added dotenv to the project so later we should be able to use `process.env.MONGO_DB_URI` instead of hardcoding the uri here

Reply:
github may actually yell at you for committing this